packages/emotion/src/getTheme.ts

```diff
@@ -82,6 +81,8 @@ const getTheme =
           'No theme provided for [InstUISettingsProvider], using default `canvas` theme.'
         )
       }
+      //TODO: replace cloneDeeps with native `structuredClone` API
+      //once we hit last 2 version browser support
```

Comment:
Can you please make a ticket about this, so we dont forget it?

Reply:
yeah, sure, good idea